### ISSUE TITLE
Inovative solution

### ISSUE THREAD

**Issue by saitejach127** [WebdevteamIG/blockchainvoting#10]
Proposing a new way to conduct elections 
Propose a new architecture that is secure and fast to conduct elections

**Comment by sahil-ramagiri** [comment]
I am trying to understand this, and I have some doubts, 

1. I see that anyone can vote, so what stops people outside the nation or who don't have the right to vote from voting.
2. The election can be stopped by admin at any time, usually voting is for a fixed duration, so we can pass it along constructor and will be fixed.
3. The candidates are not fixed too, the admin can add any new candidates at any time, also he could miss some votes by the time his name is added.

**Comment by saitejach127** [comment]
Yeah, those are some good things to point out. Do you think you can propose a well-architected solution for fast and also less memory consumption.

**Comment by sahil-ramagiri** [comment]
Currently I don't know enough to do that, but I will learn and try to think any.

**Comment by saitejach127** [comment]
No it's not about code, your issues seem good can you propose a step by step workflow on how it should be conducted so that we can add that as an issue and others or you can code on it.
